app/services/planning_comparison_service.rb

```diff
@@ -126,7 +123,7 @@ def self.resolve_projects(work_packages)
 
   def self.resolve_types(work_packages)
     type_ids  = work_packages.map(&:type_id).uniq.compact
-    types     = Hash[Type.find(type_ids).map { |type| [type.id, type] }]
+    types     = Hash[::Type.find(type_ids).map { |type| [type.id, type] }]
```

Comment:
Useless assignment to variable - `types`. Did you mean `type_ids`?<br>Unnecessary spacing detected.